easybuild/framework/extension.py

```diff
@@ -138,6 +138,12 @@ def __init__(self, mself, ext, extra_params=None):
                 self.log.debug("Skipping unknown custom easyconfig parameter '%s' for extension %s/%s: %s",
                                key, name, version, value)
 
+        self.cfg['parallel'] = self.master.orig_parallel
+        par = get_parallel_ec_param_value(self.cfg, self.log)
+        self.log.info("Setting parallelism: %s", par)
```

Comment:
Redundant log.info line

Reply:
Actually both are redundant as `get_parallel_ec_param_value` does the logging and the log already has the context of the extension (a bit above though, but still)